pkg/minishift/docker/image/sysproc_darwin.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
```

Comment:
hmm, I think there is a case to be made for this being the right location. However, I have a feeling I will need this shortly somewhere else. Moving into _pkg/util/os/process_